python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -183,14 +183,26 @@ jitAndCreateArgs(const std::string &name, MlirModule module,
     rawArgs = nullptr;
     size = argsCreator(runtimeArgs.data(), &rawArgs);
   }
-  return std::make_tuple(jit, rawArgs, size);
+
+  std::size_t returnOffset = 0;
+  if (runtimeArgs.size()) {
+    auto expectedPtr = jit->lookup(name + ".returnOffset");
+    if (!expectedPtr) {
+      throw std::runtime_error(
+          "cudaq::builder failed to get returnOffset function.");
+    }
+    auto returnOffsetCreator =
+        reinterpret_cast<std::size_t (*)()>(*expectedPtr);
+    returnOffset = returnOffsetCreator();
```

Comment:
Do you want to check for NoReturnOffset being returned here?

Reply:
Do you know what is returned in the case or error?

Reply:
Got it, updated!